thetis/coupled_timeintegrator_2d.py

```diff
@@ -92,11 +92,7 @@ def _create_tracer_integrator(self):
                                                                   semi_implicit=self.options.timestepper_options.use_semi_implicit_linearization,
                                                                   theta=self.options.timestepper_options.implicitness_theta)
             else:
-                self.timesteppers.tracer = self.tracer_integrator(solver.eq_tracer,
```

Comment:
Do we have to remove this structure altogether? I assumed that this the error is a temporary fix until we expand this section. I have no strong feelings either way

Reply:
That's true. i could just comment it out instead if that would be better?